tests/test_asm.rs

```diff
@@ -71,7 +73,7 @@ pub struct CustomDebugger {
     pub value: Arc<AtomicU8>,
 }
 
-impl<Mac: SupportMachine> Debugger<Mac> for CustomDebugger {
+impl<Mac: SupportMachine> ExecutionContext<Mac> for CustomDebugger {
```

Comment:
Similarly, if we really care about API compatibiliy, we should preserve the original Debugger trait, and have Debugger trait implement ExecutionContext.

Reply:
OK. Will do.

Reply:
It's similarly not possible to

```rust
impl<Mac, T> ExecutionContext<Mac> for T
where
    Mac: SupportMachine,
    T: Debugger<Mac>,
```

due to conflict implementations.